pallets/rc-migrator/src/lib.rs

```diff
@@ -1482,7 +1534,12 @@ pub mod pallet {
 
 			if new == MigrationStage::Starting {
 				defensive_assert!(
-					matches!(old, MigrationStage::WaitingForAh | MigrationStage::Scheduled { .. }),
+					matches!(
+						old,
+						MigrationStage::WaitingForAh { .. } |
+							MigrationStage::Scheduled { .. } |
+							MigrationStage::CoolOff { .. }
+					),
```

Comment:
Please also check the `is_ongoing` of the Stage enum, but i think it does not need modifications.

Reply:
yes, it looks correct

``` rust
	pub fn is_ongoing(&self) -> bool {
		!matches!(
			self,
			MigrationStage::Pending |
				MigrationStage::Scheduled { .. } |
				MigrationStage::MigrationDone
		)
	}
```